curriculum/js2/1.js

```diff
@@ -5,8 +5,14 @@
  * @returns {array}
  */
 
-const solution = (a) => {
-  return [];
+const solution = (a,i=0) => {
```

Comment:
I keep on seeing this file being committed as a change.

Maybe next time you can just branch of a dev branch or master branch.